### PR TITLE
New package: trax

### PR DESCRIPTION
**Pull request by mjambon**

### CODE REVIEW

**Comment by UnixJunkie** [comment]
There are no install / uninstall targets in the opam file. Is this OK?

**Comment by mjambon** [comment]
@UnixJunkie jbuilder produces a file `trax.install` that contains the list of files to install.
```
$ cat trax.install 
lib: [
  "_build/install/default/lib/trax/META" {"META"}
  "_build/install/default/lib/trax/opam" {"opam"}
  "_build/install/default/lib/trax/trax.cmi"
  "_build/install/default/lib/trax/trax.cmx"
  "_build/install/default/lib/trax/trax.cmt"
  "_build/install/default/lib/trax/trax.cmti"
  "_build/install/default/lib/trax/trax.mli"
  "_build/install/default/lib/trax/trax.cma"
  "_build/install/default/lib/trax/trax.cmxa"
  "_build/install/default/lib/trax/trax.a"
  "_build/install/default/lib/trax/trax.cmxs"
]
doc: [
  "_build/install/default/doc/trax/README.md"
  "_build/install/default/doc/trax/LICENSE"
]
```

File: packages/trax/trax.0.3.0/descr [hunk 1]
```diff
@@ -0,0 +1 @@
+Trax defines a special exception, which is used to store a trace of where the exception was raised and re-raised, independently from the state of the call stack.
```
Comment (avsm) [inline packages/trax/trax.0.3.0/descr]: The first line is used as a short description -- would it be possible to have a shorter description of trax on the first line? thanks!

Comment (mjambon) [inline packages/trax/trax.0.3.0/descr]: Thanks. The description should look better now.

I still don't understand Github's review process. Am I supposed to do something to let you know that I'm done with my changes or do you have to guess?

Comment (UnixJunkie) [inline packages/trax/trax.0.3.0/descr]: It's better to give some feedback after your commits, so that people notice something happened.

**Comment by avsm** [comment]
Thanks!

**Pull request merged**